### PR TITLE
refresh to v3.2.0

### PR DESCRIPTION
**Pull request by ritumd**
## Checklist before submitting

- [X] Did you read the [contributor guide](https://github.com/open-ce/open-ce/blob/main/CONTRIBUTING.md)?
- [ ] Did you update any affected [documentation](https://github.com/open-ce/open-ce/blob/main/doc/)?
- [ ] Did you write any tests to validate this change?

## Description

- Refresh to v3.2.0. 
- Skipping the "test_get_third_party_dependencies" as it is presently failing.
- Deleted recipe/run_test.py as the tests were executing twice - once through run_test.py and once through the command in 
the meta.yaml

## Review process to land 

1. All tests and other checks must succeed.
2. At least one [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) must review and approve.
3. If any  [maintainer](https://github.com/open-ce/open-ce/blob/main/MAINTAINERS.md) requests changes, they must be addressed.

### CODE REVIEW

**Review by alexlang74 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -52,19 +52,19 @@ requirements:
...
     - mock
-    - importlib_metadata ==2.0.0
     - hypothesis
   imports:
     - spacy
   commands:
-    - python -m pytest --tb=native --pyargs {{ name }}
-
+    - python -m pytest --tb=native -k "not test_get_third_party_dependencies" --pyargs {{ name }} 
+
```
Comment (cdeepali) [inline recipe/meta.yaml:67]: We can remove this whitespace.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -52,19 +52,19 @@ requirements:
     - packaging >=20.0
     - typing_extensions {{ typing_extensions }}  # [py<38] 
     - jinja2
+    - langcodes
```
Comment (npanpaliya) [inline recipe/meta.yaml:55]: Do we not need any pin for langcodes?

Comment (ritumd) [reply recipe/meta.yaml:55]: langcodes is used only by spacy and is built by us, so not added pin for it in spacy recipe

File: recipe/meta.yaml (hunk 1 above)
Comment (ritumd) [reply recipe/meta.yaml:67]: ok

**Review by npanpaliya — APPROVED** [review]

**Pull request merged**